netlogo-gui/src/test/app/infotab/InfoFormatterTests.scala

```diff
@@ -104,13 +105,13 @@ class InfoFormatterTests extends FunSuite {
 
   // in contrast, here no <pre> because no blank line
   test("fenced code blocks 2") {
-    assertResult("""|<p>foo<br/><code>
-              |   bar
-              |  baz
-              |qux
-              |</code><br/>oof</p>""".stripMargin) {
+    assertResult("""|<p>foo<br/><code class="text">
```

Comment:
This test fails because, I think the missing newlines confuses pegdown and the block is handled as code (so we get all kind of `font` tags from the conversion to HTML).

But I don't think this should be supported anyway, you're supposed to put a newline before all sorts of things, including code blocks.